### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by guillaumerose**

### CODE REVIEW

File: pkg/tap/switch.go [hunk 1]
```diff
@@ -196,6 +196,9 @@ loop:
 		if err != nil {
 			return errors.Wrap(err, "cannot read size from socket")
 		}
+		if n != len(sizeBuf) {
+			return fmt.Errorf("unexpected size %d != %d", n, len(sizeBuf))
+		}
```
Comment (cfergeau) [inline pkg/tap/switch.go:201]: What potential issue is this fixing?

https://golang.org/pkg/io/#ReadFull says:

> On return, n == len(buf) if and only if err == nil.

So this looks like a false positive in the linter you used?

Comment (guillaumerose) [reply pkg/tap/switch.go]: yes I removed this part

**Comment by cfergeau** [comment]
Totally forgot about this PR, and redid the same work as part of https://github.com/containers/gvisor-tap-vsock/pull/34
This can be closed.

**Comment by guillaumerose** [comment]
yes

**Pull request closed without merging**